static/js/index.js

```diff
@@ -6,14 +6,16 @@ function change_to_reset_form () {
 
     $(document).on('click', '#change-to-reset-password-form', function () {
         $.ajax({
-            url: 'reset-password-handler/',
+            url: '/login/change-to-reset-form/',
             type: 'POST',
             data: {
                 action: 'reset-password',
                 csrfmiddlewaretoken: Cookies.get('csrftoken'),
             },
             success: function(response){
-                $('#main-form').html(response)
+                if (response['status'] === 'ok') {
+                    $('#main-form').html(response['content']);
+                }
```

Comment:
Great! I also wanted to do this!

Reply:
But wouldn't it better, if you would send an already rendered fail and/or success html-response, which I'm going to add?